api/logic/job_janitor.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	JOB_LIFETIME           = time.Hour * 24
-	JANITOR_SLEEP_DURATION = time.Hour * 1
+	JANITOR_SLEEP_DURATION = time.Minute * 10
```

Comment:
note we bump up the job janitor to try and cleanup bursts of activity. every 10 minutes may be high, we could probably set it to 30m